Description: Fix elocation.pc pkgconfig file generation
 Variable needs to be filled in with Requires.private
Author: Ross Vandegrift <ross@kallisti.us>
Forwarded: no
Last-Update: 2015-09-27
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/pc/elocation.pc.in
+++ b/pc/elocation.pc.in
@@ -5,7 +5,7 @@
 
 Name: elocation
 Description: Enlightenment location library
-@pkgconfig_requires_private@: @requirements_elocation@
+Requires.private: @requirements_pc_elocation@
 Version: @VERSION@
 Libs: -L${libdir} -lelocation
 Libs.private: -lm
